src/hotspot/cpu/aarch64/aarch64_sve_ad.m4

```diff
@@ -221,12 +221,12 @@ source %{
       case Op_ExtractC:
       case Op_ExtractUB:
       // Vector API specific
+        return false;
       case Op_StoreVectorScatter:
-      // Currently the implementation for partial vectors are not implemented yet.
-      // Will add them in a separate patch.
+       // Currently the implementation for partial vectors are not implemented yet.
+       // Will add them in a separate patch.
         return vlen * type2aelembytes(bt) == MaxVectorSize;
       case Op_VectorLoadConst:
-        return false;
```

Comment:
VectorLoadConst is not implemented yet, isn't it?

Reply:
It's my fault. I have fixed that. Thank you for your review.